File: app/src/main/java/ensan/patientapp/view/activity/home/model/Datum.java

package ensan.patientapp.view.activity.home.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class Datum {

    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String enName;
    @SerializedName("ar_name")
    @Expose
    private String arName;
    @SerializedName("image")
    @Expose
    private String image;

    private boolean isChecked = false;

    public boolean isChecked() {
        return isChecked;
    }

    public void setChecked(boolean checked) {
        isChecked = checked;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getEnName() {
        return enName;
    }

    public void setEnName(String enName) {
        this.enName = enName;
    }

    public String getArName() {
        return arName;
    }

    public void setArName(String arName) {
        this.arName = arName;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

}

File: app/src/main/java/ensan/patientapp/view/fragment/wallet/model/Datum.kt
package ensan.patientapp.view.fragment.wallet.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Datum {

    @SerializedName("amount")
    @Expose
    var amount: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("date")
    @Expose
    var date: String? = null

    @SerializedName("type")
    @Expose
    var type: String? = null

    @SerializedName("invoice_no")
    @Expose
    var invoiceNo: String? = null

    @SerializedName("booking_id")
    @Expose
    var bookingId: String? = null
}